tests/conftest.py

```diff
@@ -10,6 +10,23 @@
     from py._path.local import LocalPath
 
 
+@pytest.fixture(autouse=True)
+def temporarily_delete_setupcfg(tmpdir: "LocalPath") -> Iterator[None]:
+    """
+    Temporarily delete setup.cfg so it can be recreated during tests.
+
+    Parameters
+    ----------
+    tmpdir
+        Pytest fixture, gives us a temporary directory.
+    """
+    filename = Path("setup.cfg")
+    temp_file = Path(tmpdir) / "setup.cfg"
```

Comment:
`temp_file = Path(tmpdir) / filename` so that setup.cfg is hard coded at only one place.